src/lib_ccx/params_dump.c

```diff
@@ -11,8 +11,9 @@ void params_dump(struct lib_ccx_ctx *ctx)
 	switch (ccx_options.input_source)
 	{
 		case CCX_DS_FILE:
+			mprint("Files (%d): ", ctx->num_input_files);
 			for (int i=0;i<ctx->num_input_files;i++)
-				mprint ("%s%s",ctx->inputfile[i],i==(ctx->num_input_files-1)?"":",");
+				mprint ("%s%s",ctx->inputfile[i],i==(ctx->num_input_files-1)?"":", ");
```

Comment:
This seems more debug that general, maybe mprint() is too much?

Reply:
What would you suggest? :)

Reply:
What do we use for debug messages? :-)

Reply:
@cfsmp3 I changed this (https://github.com/CCExtractor/ccextractor/pull/994/files#diff-20a92dffa6342caa6a83fd990812e4cbR14) to debug message. Should I change the current line (L16) as well? It was already in mprint though, and I think having the ability to see the files being processed is fine without running it in debug mode.